pkg/apis/batch/v1/conversion.go

```diff
@@ -26,21 +26,35 @@ import (
 )
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
-	return scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("Job"),
+	err := scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("Job"),
 		func(label, value string) (string, string, error) {
 			switch label {
 			case "metadata.name", "metadata.namespace", "status.successful":
 				return label, value, nil
 			default:
-				return "", "", fmt.Errorf("field label %q not supported for batchv1.Job", label)
+				return "", "", fmt.Errorf("field label %q not supported for Job", label)
 			}
-		},
-	)
+		})
+	if err != nil {
+		return err
+	}
+	err = scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("CronJob"),
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "metadata.name", "metadata.namespace":
```

Comment:
if you're just adding the default metadata conversions, you don't need to register this... those are supported automatically

Reply:
Removed.